src/main/java/org/apache/fineract/operations/TransactionRequest.java

```diff
@@ -76,6 +76,10 @@ public class TransactionRequest extends AbstractPersistableCustom<Long> {
 
     @Column(name = "EXTERNAL_ID")
     private String externalId;
+
+    @Column(name = "CLIENTCORRELATIONID")
```

Comment:
use the same format in names as CLIENTCORRELATION_ID

Reply:
Based on this PR https://github.com/openMF/ph-ee-operations-app/pull/43/files that is merged, if i change the value here i would need to update it all places, should i do it?

Reply:
lets keep it this way as there is a dependency now.